### PR TITLE
added example for isometric map

### PR DESCRIPTION
**Pull request by YegorStolyarov**
#1

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,6 +27,13 @@ seldom_fn_plugin = { version = "0.4", optional = true }
 seldom_interop = { version = "0.4", optional = true }
 seldom_state = { version = "0.7", optional = true }
 
+# crates.io version is still bugged, so take it from github
```
Comment (Seldom-SE) [inline Cargo.toml]: What's the bug?

**Comment by Seldom-SE** [comment]
The map is showing for me on Windows, but not Linux. I suspect it's a bug from `bevy_ecs_tilemap`. I'll look into it when I have more time, likely this weekend

**Comment by YegorStolyarov** [comment]
> The map is showing for me on Windows, but not Linux. I suspect it's a bug from `bevy_ecs_tilemap`. I'll look into it when I have more time, likely this weekend

Thats strange, because I developed it on linux. Feel free to ask for any details.
image
image

**Comment by Seldom-SE** [comment]
I fixed it by running `cargo update` lol

File: Cargo.toml (hunk 1 above)
Comment (YegorStolyarov) [reply Cargo.toml]: when changing ecs_map version to 0.11.0, I receiving this error on running example:
image

when getting it from github, everything works fine.
Probably the bug which occurs with crates.io version:
https://github.com/StarArawn/bevy_ecs_tilemap/issues/448

**Comment by YegorStolyarov** [comment]
@Seldom-SE Feel free to ask for any required updates :)

**Comment by Seldom-SE** [comment]
I'm coming back from being really busy, so I think I'll get back to this somewhat soon

**Comment by Seldom-SE** [comment]
Oh, I forgot that this was pretty much ready. Good work, thanks for the pr

**Pull request merged**